pkg/virt-controller/watch/vmi_test.go

```diff
@@ -1180,14 +1183,701 @@ var _ = Describe("VirtualMachineInstance watcher", func() {
 			table.Entry("and in failed state", k8sv1.PodFailed),
 		)
 	})
+
+	Context("hotplug volume", func() {
```

Comment:
we need at least one set of test cases that validate the top level ```contoller.Execute()``` flow.  This ensures the expected API calls are invoked, like creation/deletion of attachmentPods as well as updates to the VMI status.